boa3/model/builtin/interop/contract/__init__.py

```diff
@@ -3,10 +3,12 @@
            'DestroyMethod',
            'GasProperty',
            'NeoProperty',
-           'UpdateMethod'
+           'UpdateMethod',
+           'ContractType'
```

Comment:
I wasn't going to request a change because of this, but as there are other things to change I am including a note here anyway
Try to keep the imports dictionaries in the lexicographic order